gradle.properties

```diff
@@ -1,2 +1,3 @@
-org.gradle.daemon=false
```

Comment:
i could not find a good argument for having this (either in jira or in git log). I did not see any problem (yet).